### PR TITLE
Various bug fixes

### PR DESCRIPTION
**Pull request by dephillipsmichael**
These are bug fixes found during E2E testing, I will add some comments to the code review to point out the history of each.

### CODE REVIEW

File: DataMigration/src/main/java/org/sagebionetworks/dian/datamigration/tools/schedulev2/ScheduleV2Migration.java [hunk 1]
```diff
@@ -60,58 +63,65 @@ public static void runV2Migration() throws IOException {
...
-
-                System.out.println("Performing V2 migration on " + arcId);
-
-                // Check for a user that has already migrated to V2 and signed into the app
-                SageUserClientData clientData = SageUserClientData.Companion.fromStudyParticipant(gson, p);
-                boolean hasMigrated = SageUserClientData.Companion.hasMigrated(clientData);
-
-                if (!hasMigrated) {
-                    System.out.println("Creating schedule for " + arcId);
-                    createV2Schedule(v1Schedule, uId, sId);
```
Comment (dephillipsmichael) [inline DataMigration/src/main/java/org/sagebionetworks/dian/datamigration/tools/schedulev2/ScheduleV2Migration.java:94]: I wrapped each participant's migration in a try/catch where errors are collected and then printed at the end of the program execution.  This allows for the migration to fully operate even if there is a single user error.

File: DataMigration/src/main/java/org/sagebionetworks/dian/datamigration/tools/schedulev2/ScheduleV2Migration.java [hunk 2]
```diff
@@ -183,31 +193,30 @@ public static TestSchedule createHMSchedule(String uId, String scheduleJson) thr
...
         for (int i = 0; i < v1Schedule.getStudyBursts().size(); i++) {
             SageV1StudyBurst studyBurst = v1Schedule.getStudyBursts().get(i);
 
-            // V1 HM scheduling had the practice (baseline) test included and set to
-            // the first study burst start date. This shifted all study bursts by 1 day.
             // The new V2 Sage scheduling does not include the practice test, and so
             // the study as a whole, and the first study burst schedule starts.
             DateTime startDate = SageScheduleController.Companion
-                    .createDateTime(studyBurst.getStartDate()).plusDays(1);
+                    .createDateTime(studyBurst.getStartDate());
```
Comment (dephillipsmichael) [inline DataMigration/src/main/java/org/sagebionetworks/dian/datamigration/tools/schedulev2/ScheduleV2Migration.java:205]: We originally had the schedule on a day delay, but that has changed and the day delay is no longer needed.

File: DataMigration/src/main/java/org/sagebionetworks/dian/datamigration/tools/translation/LocaleResource.java [hunk 3]
```diff
@@ -38,6 +38,9 @@ public LocaleResource(Map<String,String> map) {
 
         if(countryKey.equals("US") && languageKey.equals("en")) {
             resource = "values" + File.separator;
+        } else if (countryKey.equals("419") && languageKey.equals("es")) {
```
Comment (dephillipsmichael) [inline DataMigration/src/main/java/org/sagebionetworks/dian/datamigration/tools/translation/LocaleResource.java:41]: This is an unrelated change to the translation tool, but we just added Latin American Spanish to the INV Arc app, and this enables it for Android/iOS

**Review by nategbrown9 — APPROVED** [review]
Looks good.

**Pull request merged**